draft-ietf-taps-impl.md

```diff
@@ -556,10 +556,19 @@ not using TCP), the action should fail gracefully. The application may be inform
 The Transport Services implementation should allow protocol instances in the Protocol Stack to pass up arbitrary generic or protocol-specific
 errors that can be delivered to the application as Soft Errors. These allow the application to be informed of ICMP errors, and other similar events.
 
+## Pooled Connection {#pooled-connections}
+
+For protocols that employ request/response pairs and do not require in-order delivery of the responses, like HTTP/2, the transport implementation may distribute interactions across several underlying transport connections. 
```

Comment:
HTTP/2 is not a good case to use multiple underlying TCP/TLS connections... we generally want to multiplex. Or is this just meaning that the HTTP/2 streams themselves are a pool?

Reply:
I only thought about the HTTP/2 Interface, not about the protocol itself. Perhaps it would be better to speak of HTTP without version. 
Yes, we want to multiplex, but maybe we want to multiplex over several connections, e.g., using different paths or interfaces (there are other reasons like getting twice the TCP share do not want to advocate for).

Reply:
Yes, that change I think does help!